### PR TITLE
fix #55

### LINKED ISSUE #55: "updated _ ago" is not updated correctly
## Describe the bug
After updating a Skapp, the little text bellow does not update correctly
In my example it says "updated a month ago" even though I have just updated the skapp.

### PR DESCRIPTION
**Pull request by Delivator**
# PULL REQUEST

## Overview
Fix "updated _ ago" in the Dapp card not updated correctly
Fixed by using the `addedOn` of the last item added to the `skylinks` list

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [ ] All new methods or updated methods have clear docstrings
 - [ ] Testing added or updated for new methods
 - [ ] Verify if any changes impact the WebPortal Health Checks
 - [ ] Approriate documentation updated
 - [ ] Changelog file created

## Issues Closed
#55

### CODE REVIEW

**Comment by Delivator** [comment]
Any update? Its a simple fix, should not break anything

File: src/components/DappCard.js [hunk 1]
```diff
@@ -48,7 +48,8 @@ export default function DappCard({ dapp, actions = true }) {
           {dapp.addedOn && (
```
Comment (m-cat) [inline src/components/DappCard.js:48]: Not familiar with this codebase and haven't looked into the bug, but can you please explain why was this not working? Looks like `addedOn` is defined, based on line 48.

**Review by kwypchlo — APPROVED** [review]

File: src/components/DappCard.js (hunk 1 above)
Comment (Delivator) [reply src/components/DappCard.js:48]: AFAIK addedOn is set once when you first add a dapp to the list
So it should be the same as `dapp.skylinks[0]`
Getting the last item in that list should give you the date on which it was last updated

**Comment by Delivator** [comment]
Check failed because at deploy (siasky.net network error) not because it had a build error

File: src/components/DappCard.js [hunk 2]
```diff
@@ -48,7 +48,8 @@ export default function DappCard({ dapp, actions = true }) {
...
             <p className="text-palette-300 text-xs flex items-center flex-row truncate">
               <ClockIcon className="w-4 inline-flex mr-1 flex-shrink-0" />{" "}
-              {dapp.skylinks.length > 1 ? "updated" : "installed"} <RelativeDate date={dapp.addedOn} />
+              {dapp.skylinks.length > 1 ? "updated" : "installed"}
+              <RelativeDate date={dapp.skylinks[dapp.skylinks.length - 1].addedOn} />
```
Comment (kwypchlo) [inline src/components/DappCard.js:52]: You're missing a space between the label and the relative time now.

```suggestion
              {dapp.skylinks.length > 1 ? "updated" : "installed"}{" "}
              <RelativeDate date={dapp.skylinks[dapp.skylinks.length - 1].addedOn} />
```

Screenshot 2021-12-20 at 17 45 53
Screenshot 2021-12-20 at 17 45 29

**Review by kwypchlo — CHANGES_REQUESTED** [review]

File: src/components/DappCard.js (hunk 2 above)
Comment (Delivator) [reply src/components/DappCard.js:52]: Oh thats what that char was for 😅 I though that I added that by accident

**Review by m-cat — APPROVED** [review]

File: src/components/DappCard.js (hunk 1 above)
Comment (m-cat) [reply src/components/DappCard.js:48]: Ah gotcha, that makes sense.

**Review by kwypchlo — APPROVED** [review]

**Comment by kwypchlo** [comment]
The base branch requires all commits to be signed. [Learn more about signing commits.](https://docs.github.com/articles/about-gpg/)

**Comment by Delivator** [comment]
Do I need to sign them or can you do that too?

**Comment by kwypchlo** [comment]
> Do I need to sign them or can you do that too?

I can't do that for you, you will need to set up commits signing yourself and rebase the pull request so that the commits are rewrtitten I think. If this is too much I can just create new pull request myself with your change. Let me know.

**Comment by Delivator** [comment]
Okay I'm not quiet sure how to rebase this PR, but I now have setup git to sign my commits from here on.
I think for this PR it'd be easier if you just apply the changes manually/in a new pr and close this one.

**Pull request closed without merging**